### PR TITLE
Add boterinas introduction to Asterinas book

### PR DESCRIPTION
**Pull request by grief8**
This PR adds boterinas introduction to Asterinas book, which expands #1129 . See the rendering [here](https://grief8.github.io/book/to-contribute/boterinas.html).

### CODE REVIEW

**Comment by grief8** [comment]
@boterinas claim

File: docs/src/to-contribute/boterinas.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  *Example:* `@boterinas release-assignment`
+  
+- **`@boterinas assign @user`**  
+  Assigns a specific user to the issue or pull request. Only team members have permission to assign other users.  
+  *Example:* `@boterinas assign @some_asterinas`
+
+### Label Management
+- **`@boterinas label <label>`**  
+  Adds a label to the issue or pull request.  
+  *Example:* `@boterinas label C-bug`
```
Comment (tatetian) [inline docs/src/to-contribute/boterinas.md:38]: Most examples are useless: they just repeat the command format. The examples for the `label` command is probably the only ones that provide some extra information. I suggest removing useless examples and only the useful ones.

File: docs/src/to-contribute/boterinas.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Bot for Asterinas
```
Comment (tatetian) [inline docs/src/to-contribute/boterinas.md:1]: Remove "Asterinas" from the title. Every page of the Asterinas book is for Asterinas. So mentioning Asterinas in the title provides little information. I think good candidate names for this page are "Boterinas" and "Bot for Github".

File: docs/src/to-contribute/boterinas.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# Bot for Asterinas 
+
+## Introduction
+
+`@boterinas` is a general-purpose bot designed for a wide variety of tasks in Asterinas. It streamlines maintenance tasks to enhance workflow efficiency. 
+
+Commands are issued by writing comments that start with the text `@boterinas`. The available commands depend on which repository you are using. The main Asterinas repository contains a `triagebot.toml` file where you can see which features are enabled. 
+
+Commands for GitHub issues or pull requests should be issued by writing `@boterinas` followed by the command anywhere in the comment. Note that `@boterinas` will ignore commands in markdown code blocks, inline code spans, or blockquotes. You can enter multiple `@boterinas` commands in a single comment.
```
Comment (tatetian) [inline docs/src/to-contribute/boterinas.md:9]: markdown --> Markdown

**Review by tatetian — COMMENTED** [review]
Excellent job on writing up the doc of Boterinas!

File: docs/src/to-contribute/boterinas.md (hunk 1 above)
Comment (grief8) [reply docs/src/to-contribute/boterinas.md:38]: Get it. I will replace the example with informative ones, such as `@boterinas label C-enhancement C-rfc` to show multiple labeling in one command.

File: docs/src/to-contribute/boterinas.md (hunk 2 above)
Comment (grief8) [reply docs/src/to-contribute/boterinas.md:1]: I prefer "Boterinas"~

**Review by tatetian — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**